libpod/image/prune.go

```diff
@@ -0,0 +1,35 @@
+package image
+
+import "fmt"
+
+// GetUnamedImages returns a slice of images that have no names/unused
+func (ir *Runtime) GetUnamedImages() ([]*Image, error) {
+	var (
+		unamedImages []*Image
+	)
+	allImages, err := ir.GetImages()
+	if err != nil {
+		return nil, err
+	}
+	for _, i := range allImages {
+		if len(i.Names()) == 0 {
```

Comment:
`Image.Remove`, when removing parents of an image, also does not remove images that have children.  Shouldn't this function and `Image.Remove` use the same criteria for removing unnecessary images?

(I’m much less sure _which_ of the criteria :)  In my mental model the children condition does not fit, but I haven’t looked into this deeply.)

Reply:
I think this logic is OK.